### PR TITLE
Adding support for IBM POWER(ppc64le)

### PR DESCRIPTION
**Pull request by bahetiamit**
Signed-off-by: Amit Baheti aramswar@in.ibm.com

### CODE REVIEW

File: build/setup-operator-sdk.sh [hunk 1]
```diff
@@ -9,8 +9,11 @@ if ! [ -e "./operator-sdk" ]; then
     *)          file=operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu;;
   esac
 
+ARCH=$(uname -m)
```
Comment (jmesnil) [inline build/setup-operator-sdk.sh:12]: Could you please apply the same change to build/setup-operator-sdk-e2e-tests.sh?

We use a different version of the operator-sdk for the end-to-end tests but we need to make sure this version also uses the proper arch.

Comment (bahetiamit) [reply build/setup-operator-sdk.sh:12]: Yes @jmesnil . I have made the required changes.

**Comment by jmesnil** [comment]
@bahetiamit thanks

**Pull request merged**